browser/ui/views/tabs/brave_compound_tab_container.cc

```diff
@@ -138,6 +138,7 @@ void BraveCompoundTabContainer::SetScrollEnabled(bool enabled) {
 
   if (enabled) {
     scroll_view_ = AddChildView(std::make_unique<CustomScrollView>());
+    scroll_view_->SetBackgroundThemeColorId(kColorToolbar);
```

Comment:
Right - the scroll view is also painted to an (opaque?) layer, so needs all pixels painted. Previously, the tab container would paint the background for it, but since we're moving background painting into `BraveTabStrip` (which is above this scroll view), we need to have the scroll view also take care of background painting.

Maybe it would be easier to have the tab container paint to a layer instead of `BraveTabStrip`. I'm not sure if I tried that, but I'll try it again.

Reply:
`BraveTabStrip` needs to be the layer-based ancestor for tabs because tabs are moved out of `TabContainer` and into `TabDragContext` when dragging. `BraveTabStrip` is the parent for both.